apis/betterself/v1/events/tests.py

```diff
@@ -348,12 +348,27 @@ def test_that_rolling_window_parameter_works(self):
 
         # sum up the two individual results to make sure the analytics is correct
         very_productive_time_list = DailyProductivityLog.objects.filter(user=self.default_user, date__lte=five_days_ago,
-                                                                        date__gte=six_days_ago).values_list(
+            date__gte=six_days_ago).values_list(
```

Comment:
Probably look into YAPF and just figure out the style and run it on this entire thing ...